h2o-algos/src/main/java/hex/modelselection/ModelSelectionUtils.java

```diff
@@ -210,6 +605,12 @@ public static int findBestR2Model(double lastBestR2, GLMModel[] bestR2Models) {
         return currBestR2 > lastBestR2 ? bestIndex : -1;
     }
     
+    public static int findBestMSEModel(double lastMinMSE, ModelSelection.SweepModel[] currModels) {
+        double currMinMSE = Stream.of(currModels).map(x->x._errorVariance).min(Comparator.naturalOrder()).get();
+        int bestIndex = Stream.of(currModels).map(x->x._errorVariance).collect(Collectors.toList()).indexOf(currMinMSE);
```

Comment:
Since this is called from the `while (true)`, I wonder if it could be more efficient. I don't know much about efficiency in java but I'd assume going through the currModels just once would be better.

I didn't find `argmin` in stream api but we can use `reduce` instead. Pseudocode:
```java
int bestIndex = IntStream.range(0, currModels.length)
            .reduce((i,j) -> Comparator.naturalOrder().compare(currModels[i]._errorVariance, currModels[j]._errorVariance) <= 0 ? i : j).getOrElse(-1);
double currMinMSE = currModels[bestIndex]._errorVariance;
```

Feel free to ignore this suggestion if currModels is expected to be too small to make a difference in both approaches or if you think my way would be slower or less legible.